### ISSUE TITLE
Interception method execution result

### ISSUE THREAD

**Issue by mspectr** [Virtuoze/NConcern#21]
Is this possible to capture method execution result for further modifications?

**Comment by Virtuoze** [comment]
Yes, there is a multiple way to to it, The simplest is to define an aspect with an around advisor and implements IAdvice, return value is passed by ref to allow modifications.

Télécharger Outlook pour Android<https://aka.ms/ghei36>

________________________________
